tools/importer-rest-api-specs/components/dataapigeneratorjson/templater_operations.go

```diff
@@ -59,51 +26,49 @@ func codeForOperation(operationName string, operation models.OperationDetails, r
 		sort.Ints(statusCodes)
 	}
 
-	fieldContainingPaginationDetails := ""
-	if operation.FieldContainingPaginationDetails != nil {
-		fieldContainingPaginationDetails = *operation.FieldContainingPaginationDetails
-	}
-
 	longRunning := false
 	if operation.LongRunning {
 		// TODO: we can use the `LongRunning` operation base types too
 		longRunning = true
 	}
 
-	// TODO: refactor this to use the ObjectDefinition in `./models`
-	requestObject := ObjectDefinition{}
-	if operation.RequestObject != nil {
-		requestObject.ReferenceName = operation.RequestObject.ReferenceName
-		requestObject.Type = ObjectDefinitionType(operation.RequestObject.Type)
-
-	} else if strings.EqualFold(operation.Method, "POST") || strings.EqualFold(operation.Method, "PUT") {
-		// Post and Put operations should have one but it's possible they don't
-		// TODO what goes here?
-	}
-
 	resourceId := ""
 	if operation.ResourceIdName != nil {
 		resourceId = *operation.ResourceIdName
 	}
 
-	responseObject := ObjectDefinition{}
-	if operation.ResponseObject != nil {
-		responseObject.ReferenceName = operation.ResponseObject.ReferenceName
-		responseObject.Type = ObjectDefinitionType(operation.ResponseObject.Type)
+	uriSuffix := ""
+	if operation.UriSuffix != nil {
+		uriSuffix = *operation.UriSuffix
+	}
+
+	HTTPMethod := strings.Title(strings.ToLower(operation.Method))
```

Comment:
There's enough assumptions throughout the codebase which assume this is upper-cased, we should do that here - although tbh we can inline this below, so:

```suggestion
```